Try to avoid double reverse of dx.

ScholliYT opened the pull request:
Not really working yet.

this wont work when paddle is moving "into" the ball. Maybe add "Do not collide for x seconds" Flag

ScholliYT committed:
Try to avoid double reverse of dx.

Not really working yet

ScholliYT commented:
Meine Idee war, den Ball so lange mit dx weg zu bewegen, bis der keine Intersection mehr hat. Leider funktioniert das nicht so richtig, weil der zwar für den aktuellen Frame keine Intersection mehr hat, aber dann halt direkt im nächten Frame der Bat in den Ball geschoben werden kann.

rueno committed:
Added a Booster, implemented ability to synch users points

-Added a booster which will spawn in the world occasionally. Once hit by our ball it will increase the speed of the ball by 1.
-Added synch of points for both client and host, now they just need to be rendered on a scorebord

rueno committed:
Changed Batcolors

Changed Bat colors:
-own bat is colored blue
-enemy bat is colored red

rueno commented:
Bis Montag merken, dann besprechen. Ich konnte das jetzt nicht reproduzieren.

ScholliYT commented:
Geht auch nur, wenn du den speed auf sehr low hast. Könnte sein, dass das im echten Spiel praktisch nicht passiert.

ScholliYT committed:
Comments

rueno commented:
Wie langsam hast du das denn getestet? 
![pong](https://user-images.githubusercontent.com/10176104/51348121-289e5500-1aa2-11e9-9322-fd13d0b70e77.png)
Bei der Geschwindigkeit hab ich das jetzt nicht feststellen können.

rueno committed:
Implemented a new UI for creating games

Implemented a new UI for creating games, this also implementes the functions to do a networkscan for clients and a selection of networkinterfaces for hosts (ipv4 though)

ScholliYT committed:
Refactoring + Bugfix

fixed null check bug in NetworkingDevices.java
